### PR TITLE
Update Govuk-lint and fix linting errors

### PR DESCRIPTION
**Pull request by danielburnley**
As part of the ruby upgrades govuk-lint is required to be updated

### CODE REVIEW

File: lib/tasks/backend.rake [hunk 1]
```diff
@@ -1,6 +1,7 @@
+# rubocop:disable Metrics/BlockLength
```
Comment (tijmenb) [inline lib/tasks/backend.rake]: Can we disable this for all rake tasks in `.rubocop.yml`?

File: spec/lib/router_reloader_spec.rb [hunk 2]
```diff
@@ -64,20 +64,20 @@
 
     context "error handling" do
       it "should send an exception notification, and return false on HTTP errors" do
-        r1 = stub_request(:post, "http://foo.example.com:1234/reload").to_return(status: 401, body: "Authorisation required")
-        r2 = stub_request(:post, "http://bar.example.com:4321/reload").to_return(status: 404, body: "Not found")
+        _r1 = stub_request(:post, "http://foo.example.com:1234/reload").to_return(status: 401, body: "Authorisation required")
```
Comment (tijmenb) [inline spec/lib/router_reloader_spec.rb]: I think you can remove the assignment entirely?

File: lib/tasks/backend.rake (hunk 1 above)
Comment (danielburnley) [reply lib/tasks/backend.rake]: Would that need to be updated in govuk-lint or does it pick up local `.rubocop.yml`s ?

Comment (tijmenb) [reply lib/tasks/backend.rake]: Having a local overrride works. 

Might be worth adding it to govuk-lint too (like https://github.com/alphagov/govuk-lint/pull/87), but don't wait for it to get merged.

File: lib/tasks/backend.rake [hunk 3]
```diff
@@ -46,3 +46,4 @@ namespace :backend do
     RouterReloader.reload
   end
 end
+# rubocop:enable Metrics/BlockLength
```
Comment (tijmenb) [inline lib/tasks/backend.rake]: Forgot to remove this one.

**Pull request merged**